docs-source/content/faq/oci-fss-pv.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Using OCI File Storage (FSS) for Persistent Volumes"
+date: 2020-02-12T12:12:12-05:00
+draft: false
+---
+
+If you are running your Kubernetes cluster on Oracle Container Engine 
+for Kubernetes (commonly known as OKE), and you use OCI File Storage (FSS)
+for persistent volumes to store the WebLogic domain home then the file system
+handling demonstrated in the operator persistent volume sample will require
+an update to properly initialize the file ownership on the persistent volume
+when the domain is initially created.
+
```

Comment:
Long sentence.  Maybe at least put a comma before 'then the file system'.

Reply:
Sure, @rosemarymarano, can you provide your usual input?

Reply:
BTW, this is very similar to the FAQ of OCI Load Balancer so I can update there if needed...